torchci/clickhouse_queries/number_of_force_pushes_historical/query.sql

```diff
@@ -1,13 +1,16 @@
--- !!! Query is not converted to CH syntax yet.  Delete this line when it gets converted
+-- This is used by KPI pages to get the number of force pushes
 SELECT
-    FORMAT_TIMESTAMP('%Y-%m-%d', DATE_TRUNC(:granularity, issue_comment.created)) AS bucket,
+    formatDateTime(
+        DATE_TRUNC({granularity: String }, issue_comment.created_at),
+        '%Y-%m-%d'
+    ) AS bucket,
     COUNT(DISTINCT issue_comment.issue_url) AS count
 FROM
-    commons.issue_comment
+    default .issue_comment
```

Comment:
sanity check that not putting final is intentional

Reply:
in this case, if someone edits the comment we'll end up with double, differing entries.

Since those comments are rarely edited, it can be worth leaving out if the perf hit is horrendous, otherwise perhaps we should leave it in

Reply:
Yeah, that's the same thought I have about bot comments (thus leaving out final).  The chart looks fine when comparing with HUD